glib/src/log.rs

```diff
@@ -304,30 +318,30 @@ pub fn log_default_handler(log_domain: &str, log_level: LogLevel, message: Optio
 /// ```no_run
 /// use glib::{LogLevel, g_log};
 ///
-/// g_log!("test", LogLevel::Debug, "test");
-/// g_log!("test", LogLevel::Message, "test");
+/// g_log!(Some("test"), LogLevel::Debug, "test");
```

Comment:
For the macro there should simply be another variant that works without domain so you can do

```rust
g_log!(LogLevel::Debug, "test");
g_log!("domain", LogLevel::Debug, "test");
```

Reply:
@GuillaumeGomez Can you help with that maybe?

Reply:
Taking a look!